sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3066,34 +3066,38 @@ SIREPO.app.directive('sbatchLoginModal', function() {
               '</div>',
             '</div>',
         ].join(''),
-        controller: function(requestSender, $scope) {
+        controller: function(requestSender, $scope, $rootScope, authState) {
             $scope.otp = '';
             $scope.password = '';
             $scope.username = '';
             var awaitingSendResponse = false;
             var el = $('#sbatch-login-modal');
-            var onHidden = null;
             var errorResponse = null;
+            var onHidden = null;
 
             el.on('hidden.bs.modal', function() {
-                $scope.otp = '';
-                $scope.password = '';
-                $scope.username = '';
-                $scope.sbatchLoginModalForm.$setPristine();
                 var r = {'state': 'error', 'error': 'Please try again.'};
                 if (errorResponse) {
                     r = {'state': 'error', 'error': errorResponse};
                 }
-                errorResponse = null;
+                if (authState.sbatchLoginSuccess) {
```

Comment:
I don't like persisting on authState, because it is not returned by the server directly to authState. Please create a new service for this purpose.

Reply:
https://github.com/radiasoft/sirepo/pull/2523/commits/0cb1cfd8b6624b50c619e9fccc76f0b306fc9d1a